### DISCUSSION TITLE
Sionna Load scene

### DISCUSSION THREAD

**Discussion by Advaith2212** [NVlabs/sionna#478] [General]
I tried opening sionna, when I faced this issue wherein the `scene` isn't getting imported. It was working alright till yesterday. I don't know whether this is a driver issue or not. Can you guys kindly let me help resolve this please. 

I have attached the corresponding screenshot for the same. 

 
![Screenshot from 2024-06-12 10-04-44](https://github.com/NVlabs/sionna/assets/64831500/9eda84d1-c701-4690-b6d1-4893fe09f54f)

Thanks and regards,
Advaith

**Comment by jhoydis** [comment]
Hi @Advaith2212,

This does not look like a Sionna issue but rather a problem of your driver installation.
